extension/WebJobs.Extensions.RabbitMQ/Trigger/RabbitMQListener.cs

```diff
@@ -20,6 +21,9 @@ namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
 {
     internal sealed class RabbitMQListener : IListener, IScaleMonitor<RabbitMQTriggerMetrics>
     {
+#pragma warning disable SA1000
+        private static readonly ActivitySource ActivitySource = new("Microsoft.Azure.WebJobs.Extensions.RabbitMQ");
```

Comment:
Use class name with new instead of disabling the style check.

Reply:
Doing that makes C# analyzer recommend the latest syntax, so there is no syntax that can silence both analyzers. The issue is fixed in StyleCop analyzer. I will check again why we cannot have the latest StyleCop package. The last thing is to disable the warning in editorconfig but that will disable all warnings where whitespaces are not added after keywords.

Reply:
Haha, that's weird with both analyzers pointing out conflicting checks.

Reply:
It's fixed in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/3187

Reply:
Yeah, this has been fixed in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/commit/3874af7bcc5b3b79c628f824ea88c185bc5bc374

Guess the doc file just hasn't been updated.

Reply:
So StyleCop analyzer only has beta packages released since last three years. Taking dependency on the latest pre-release package should fix this problem.

Reply:
As per the PR thread, they recommend using the `1.2.xxx-beta` version of their NuGet package. Kind of annoying given the 1.2 has been beta for well over years now.

Reply:
Guess so. They really should release the official version `1.2`.